### DISCUSSION TITLE
Zeek not taking BPF

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [Security-Onion-Solutions/securityonion#2885] [Unsupported Versions] [answered]
I am having an issue adding a BPF to zeek.

In the file /opt/so/saltstack/local/pillar/global.sls  I added this to the bottom of the file

![image](https://user-images.githubusercontent.com/78620004/107085303-299b9f80-67ad-11eb-95ee-4cec60efaee7.png)

I do a salt "minionname" state.highstate and I check the bpf file on the sensor in /opt/so/conf/zeek and it looks like this

![image](https://user-images.githubusercontent.com/78620004/107085453-5d76c500-67ad-11eb-81bd-68caa4cbad8e.png)

However, zeek still sees traffic emanating from the host that should be filtered.

what am I missing????

**Comment by Acewiza** [comment]
You might need to restart something.  I just added three BPFs for my noisy local boxes like it shows in the docs with the && string, and it seems to work as expected.

**Comment by Acewiza** [comment]
Turns out, like in real life, things aren't always like they appear.  Entering a BPF as indicated actually stops ALL alerting on my system.  Tried a few variations including different ways of re-starting things and even rebooting between changes and the result was always the same:  NO ALERTING after attempting a BPF.

**Reply by oneCrazyAdmin** [reply]
I was able to get one working.  But I wrote it wrong and it only collected part of the traffic.  It almost acts like it doesn't want to exclude traffic.

**Comment by oneCrazyAdmin** [comment, marked as answer]
for whoever stumbles across this, this is what I ended up having to do

![image](https://user-images.githubusercontent.com/78620004/109086410-624ddb00-76c0-11eb-92d2-c1c3e646dbfb.png)
